### DISCUSSION TITLE
请问immortalwrt的管理界面http://immortalwrt.lan/cgi-bin/luci/为何打不开。

### DISCUSSION THREAD

**Discussion by mxzxck** [nikkinikki-org/OpenWrt-nikki#315] [Q&A] [answered]
请问启用MihomoTProxy后immortalwrt固件的默认管理界面http://immortalwrt.lan/cgi-bin/luci/为何打不开。我immortalwrt固件的路由器是http://172.16.0.254，只有通过http://172.16.0.254/cgi-bin/luci/可以访问immortalwrt固件管理界面。

**Comment by morytyann** [comment, marked as answer]
+.lan需要处理一下，添加到fake-ip-filter，并且在nameserver-policy里指向127.0.0.1

**Reply by mxzxck** [reply]
非常感谢！！！
